polaris-react/src/components/LegacyTabs/LegacyTabs.tsx

```diff
@@ -159,7 +169,11 @@ class TabsInner extends PureComponent<CombinedProps, State> {
       <div>
         <Box
           borderBlockEndWidth="1"
-          borderColor="border-subdued"
+          borderColor={
+            polarisSummerEditions2023
+              ? 'border-faint-experimental'
+              : 'border-subdued'
```

Comment:
Light suggestion to use the `<UseFeatures />` render prop here e.g.
https://github.com/Shopify/polaris/blob/f3b07cc15aef9ef069601df2b956765f02360a48/polaris-react/src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx#L258-L270